app/lib/matchmaking/calculates_total_matches.rb

```diff
@@ -0,0 +1,12 @@
+module Matchmaking
+  class CalculatesTotalMatches
+    def call(total_participants:, target_size:)
+      return 0 if total_participants < 2 || target_size < 2
+      return 1 if total_participants <= target_size
+
+      return total_participants / target_size unless target_size > 2
+
+      (total_participants / target_size.to_f).round
```

Comment:
This line should be the only thing that needs to change to change it back to `n` and `n+1` as the group sizes, but I need to remember where I put my notes about the math.